mica-rest/src/main/java/org/obiba/mica/access/rest/DataAccessRequestsResource.java

```diff
@@ -72,7 +72,7 @@ public Response create(Mica.DataAccessRequestDto dto, @Context UriInfo uriInfo)
     dataAccessRequestService.save(request);
 
     subjectAclService.addPermission("/data-access-request", "VIEW,EDIT", request.getId());
-    subjectAclService.addPermission("/data-access-request/" + request.getId() + "/comments", "ADD");
+    subjectAclService.addPermission("/data-access-request/" + request.getId() + "/comments", "ADD,VIEW,EDIT");
```

Comment:
Needed an edit (deleting) privilege for deleting DAR associated comments, I will have to check permissions per comment instance now.